src/components/listitems/CollapsibleListItems.tsx

```diff
@@ -0,0 +1,211 @@
+    iconAnimatedStyle,
+    bodyAnimatedStyle,
+    bodyInnerStyle
+  } = useAccordionAnimation({
+    defaultExpanded
+  });
+
+  const accordionBackground: IOColors = theme["appBackground-secondary"];
+  const accordionBorder: IOColors = theme["cardBorder-default"];
+
+  const onItemPress = () => {
+    toggleAccordion();
+    onToggle?.(!expanded);
+  };
+
+  const body = items.map((item, index) => (
+    <Fragment key={item.id}>
+      {index !== 0 && <Divider />}
+      {selectionEnabled ? (
+        <ListItemCheckbox
+          value={item.title}
+          description={item.description}
+          selected={selectedItemIds?.includes(item.id)}
+          onValueChange={
+            onItemSelected
+              ? selected => onItemSelected(item, selected)
+              : undefined
+          }
+        />
+      ) : (
+        <SimpleListItem value={item.title} description={item.description} />
+      )}
+    </Fragment>
+  ));
+
+  return (
+    <View
+      style={[
+        styles.accordionWrapper,
+        {
+          backgroundColor: IOColors[accordionBackground],
+          borderColor: IOColors[accordionBorder]
+        }
+      ]}
+    >
+      <TouchableWithoutFeedback
+        accessible={true}
+        accessibilityRole="button"
+        accessibilityState={{ expanded }}
+        accessibilityLabel={accessibilityLabel ?? title}
+        onPress={onItemPress}
+      >
+        <View style={styles.textContainer}>
+          <H6 color={theme["textBody-default"]}>{title}</H6>
+          <Animated.View style={iconAnimatedStyle}>
+            <Icon
+              name="chevronBottom"
+              color={theme["interactiveElem-default"]}
+            />
+          </Animated.View>
+        </View>
+      </TouchableWithoutFeedback>
+
+      <Animated.View style={bodyAnimatedStyle}>
+        <View
+          style={[bodyInnerStyle, styles.bodyInnerContainer]}
+          onLayout={onBodyLayout}
+        >
+          {body}
+        </View>
+      </Animated.View>
+
+      {/* This gradient adds a smooth end to the content. If it is missing,
+      the content will be cut sharply during the height transition. */}
+      <LinearGradient
+        style={styles.linearGradient}
+        colors={[
+          hexToRgba(IOColors[accordionBackground], 0),
+          IOColors[accordionBackground]
+        ]}
+      />
+    </View>
+  );
+};
+
+type SimpleListItemProps = {
+  value: string;
+  description?: string;
+};
+
+const SimpleListItem = ({ value, description }: SimpleListItemProps) => {
+  const theme = useIOTheme();
+
+  return (
+    <View style={IOSelectionListItemStyles.listItem}>
+      <H6 color={theme["textBody-default"]}>{value}</H6>
+      {description && (
+        <View>
+          <VSpacer size={IOSelectionListItemVisualParams.descriptionMargin} />
+          <BodySmall weight="Regular" color={theme["textBody-tertiary"]}>
+            {description}
+          </BodySmall>
+        </View>
+      )}
+    </View>
+  );
+};
```

Comment:
We already have a lot of `ListItem...` components suitable for our different needs. In this case we should use a `ListItemInfo` with a simple key-value pair. Let's discuss this during the design process to understand if it is easily replaceable.

Reply:
Addressed in 1174593452c92e2039ce91dd775365d06a459aa2